### PR TITLE
fix(database-change): sql plan task execute failed

### PR DESCRIPTION
**Pull request by kiko-art**
#### What type of PR is this?
type-bug

#### What this PR does / why we need it:
1. add log printf
2. resolve the issue with sql plan execution failure
#### Which issue(s) this PR fixes:

Fixes #

#### Additional documentation e.g., usage docs, etc.:

```docs

```

### CODE REVIEW

File: server/odc-service/src/main/java/com/oceanbase/odc/service/task/executor/task/SqlPlanTask.java [hunk 1]
```diff
@@ -426,11 +431,10 @@ private String uploadToOSS(String filePath) {
             try {
                 String objectName = cloudObjectStorageService.upload(file.getName(), file);
                 ossAddress = String.valueOf(cloudObjectStorageService.generateDownloadUrl(objectName));
-                log.info("upload sql plan task result file to OSS, file name={}", file.getName());
+                log.info("succeed to upload sql plan task result file to OSS, file download url = {}", ossAddress);
```
Comment (yizhouxw) [inline server/odc-service/src/main/java/com/oceanbase/odc/service/task/executor/task/SqlPlanTask.java:434]: seems syntax error, may 

`Upload sql plan task result to cloud object storage successfully, objectName={}`

and, downloadUrl contains ak/sk/sts, we may not output it in log content.

so as below log output.

Comment (kiko-art) [reply server/odc-service/src/main/java/com/oceanbase/odc/service/task/executor/task/SqlPlanTask.java]: Have been changed.

File: server/odc-service/src/main/java/com/oceanbase/odc/service/task/executor/task/SqlPlanTask.java [hunk 2]
```diff
@@ -162,12 +163,13 @@ protected boolean doStart(JobContext context) throws Exception {
...
         }
         result.setTotalStatements(index);
-        log.info("The sql plan task execute finished,result={}", result);
 
         // all sql execute csv file list write to zip file
         writeZipFile();
         // upload file to OSS, also contains error record where is non-null
         upload();
+
+        log.info("The sql plan task execute finished,result={}", result);
```
Comment (guowl3) [inline server/odc-service/src/main/java/com/oceanbase/odc/service/task/executor/task/SqlPlanTask.java:172]: only print the SQL that failed to execute

Comment (kiko-art) [reply server/odc-service/src/main/java/com/oceanbase/odc/service/task/executor/task/SqlPlanTask.java]: Have been changed.

**Review by yizhouxw — APPROVED** [review]
lgtm

**Review by guowl3 — APPROVED** [review]

**Pull request merged**